homeassistant/components/miele/sensor.py

```diff
@@ -440,8 +440,6 @@ async def async_setup_entry(
                         entity_class = MieleProgramIdSensor
                     case "state_program_phase":
                         entity_class = MielePhaseSensor
-                    case "state_program_type":
-                        entity_class = MieleTypeSensor
```

Comment:
Why is this removed?

Reply:
MieleTypeSensor is not needed anymore. value_fn() in MieleSensor will pick up correct native_value. Logging is handled in MieleEnum class.